src/dehtml.rs

```diff
@@ -172,15 +230,52 @@ fn dehtml_starttag_cb<B: std::io::BufRead>(
             }
         }
         "b" | "strong" => {
-            dehtml.strbuilder += "*";
+            if dehtml.get_add_text() != AddText::No {
+                dehtml.strbuilder += "*";
+            }
         }
         "i" | "em" => {
-            dehtml.strbuilder += "_";
+            if dehtml.get_add_text() != AddText::No {
+                dehtml.strbuilder += "_";
+            }
         }
         _ => {}
     }
 }
 
+fn pop_tag(count: &mut Option<i32>) {
+    if let Some(ref mut divs) = count {
+        *divs -= 1;
+        if *divs <= 0 {
+            *count = None;
+        }
+    }
+}
+
+fn maybe_push_tag(
```

Comment:
Even though this function is private, I'd document it with a reference to `Dehtml` sturcture. It is impossible to understand what the `count` argument is for without the context.